src/lsdb/catalog/association_catalog.py

```diff
@@ -1,5 +1,5 @@
 import hats as hc
-import nested_dask as nd
```

Comment:
LSDB uses pylint/isort instead of ruff, so just about all of the imports will need to be re-ordered. Should be a pretty painless `isort .`

Reply:
Gotcha, made pre-commit happy